docs/docker-integration/index.rst

```diff
@@ -0,0 +1,92 @@
+.. _docker-integration:
+
+======================
+Docker, Swarm, Compose
+======================
+
+Flocker works with Docker, Swarm and/or Compose via the Flocker Plugin for Docker.
+Follow an installation guide to get started:
+
+Flocker Installation Options
+============================
+.. raw:: html
+
+	<div class="pods-eq">
+	    <div class="pod-boxout pod-boxout--2up pod-boxout--recommended">
+		   <img src="../_images/amazon-docker2x.png" alt="Amazon AWS logo"/>
+		   <span>Use our CloudFormation template to install Flocker on AWS<em>Fastest</em></span>
+		   <a href="cloudformation.html" class="button button--fast">Install</a>
+	    </div>
+	    <div class="pod-boxout pod-boxout--2up">
+		   <img src="../_images/default2x.png" aria-hidden="true" alt=""/>
+		   <span>Install Flocker and Docker Swarm manually<em><a href="../supported.html">Works with all Supported Configurations</a></em></span>
+		   <a href="manual-install.html" class="button">Install Manually</a>
+	    </div>
+	</div>
+
+.. the following causes the toctree to be hidden on page but not in the navigation, meaning that when on the linked page, the navigation shows you where you are, which is crucial for UX.
+
+.. raw:: html
+
+   <style>
+   .toctree-wrapper { display: none; }
+   </style>
+
+.. toctree::
+
+   cloudformation
+   manual-install
+
+.. _docker-tutorials:
+
+Follow a Tutorial
```

Comment:
Maybe "Next Steps" 
The next steps would be:
 * Follow a tutorial
 * Read related articles